packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -150,6 +150,28 @@ export class W3mAccountDefaultWidget extends LitElement {
     `
   }
 
+  private orderedFeaturesTemplate() {
+    const featuresOrder = this.features?.experimental_walletFeaturesOrder || [
+      'onramp',
+      'swaps',
+      'receive',
+      'send'
+    ]
+
+    return featuresOrder.map(feature => {
+      switch (feature) {
+        case 'onramp':
+          return this.onrampTemplate()
+        case 'swaps':
+          return this.swapsTemplate()
+        case 'send':
+          return this.sendTemplate()
+        default:
+          return null
+      }
```

Comment:
Maybe we can do something like this ?

```ts
const featureTemplates = {
  onramp: () => this.onrampTemplate(),
  swaps: () => this.swapsTemplate(),
  send: () => this.sendTemplate(),
};

return featuresOrder.map((feature) => featureTemplates[feature]?.() ?? null);
```

Reply:
Literally same but we'd better to avoid using optional property access operator since it's giving possibility of not exist things - we've been recently discussing with Luka